.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/omise/omise-android
+    docker:
+      - image: circleci/android:api-28
+    environment:
+      JVM-OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - restore_cache:
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
+      - run:
+          name: Run Tests
+          command: ./gradlew lint test
+      - store_artifacts:
+          path: app/build/reports
+          destination: reports
+      - store_test_results:
+          path: app/build/test-results
```

Comment:
Could you use SDK path instead sample app path?

Reply:
As in `~/` ?

Reply:
I think that represent to root path. exactly I asking for use `sdk` instead of `app`.

Reply:
Or maybe it can add multiple path.

Reply:
Ahhhh now I understand what you mean 🤦‍♂ I'll change it